lightning/src/util/events.rs

```diff
@@ -389,6 +389,15 @@ pub enum MessageSendEvent {
 		/// The channel_update which should be sent.
 		msg: msgs::ChannelUpdate,
 	},
+	/// Used to indicate that a channel_update should be sent to a single peer.
+	/// This is used, in contrast to [`Self::BroadcastChannelUpdate`], when the channel is a
+	/// private channel and we shouldn't be informing all of our peers of channel parameters.
+	SendChannelUpdate {
```

Comment:
nit: prefer `UnicastChannelUpdate` slightly for consistency

Reply:
Hmm, all our other message events are `Send`..., though? I could move it up to sit next to the rest of the send message variants?

Reply:
Ah, true. Resolving